README.md

```diff
@@ -48,6 +48,11 @@ cd vokuro
 php composer.phar install
 ```
 
+### After instalation should set right on the directories
```

Comment:
OK so here is a suggestion:

`### After the installation, please ensure that the following folders have write permissions set`